requirements.txt

```diff
@@ -5,7 +5,7 @@ msgpack-python
 cobra==0.8.2
 cameo==0.11.0
 swiglpk==1.4.3
-gnomic
+gnomic==0.7.0
```

Comment:
I just pinned cameo 0.11.1 has gnomic pinned to 0.6.0, problem but since keeping 0.11.0 here no problem I guess